exporter/datadogexporter/examples/k8s-chart/k8s-values.yaml

```diff
@@ -0,0 +1,92 @@
+mode: daemonset
+presets:
+  kubernetesAttributes:
+    enabled: true
+extraEnvs:
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: status.podIP
+  - name: OTEL_RESOURCE_ATTRIBUTES
+    value: "k8s.pod.ip=$(POD_IP)"
+config:
+  processors:
+    k8sattributes:
+      passthrough: false
+      auth_type: "serviceAccount"
+      pod_association:
+        - sources:
+            - from: resource_attribute
+              name: k8s.pod.ip
+      extract:
+        metadata:
+          - k8s.pod.name
+          - k8s.pod.uid
+          - k8s.deployment.name
+          - k8s.node.name
+          - k8s.namespace.name
+          - k8s.pod.start_time
+          - k8s.replicaset.name
+          - k8s.replicaset.uid
+          - k8s.daemonset.name
+          - k8s.daemonset.uid
+          - k8s.job.name
+          - k8s.job.uid
+          - k8s.cronjob.name
+          - k8s.statefulset.name
+          - k8s.statefulset.uid
+          - container.image.name
+          - container.image.tag
+          - container.id
+          - k8s.container.name
+          - container.image.name
+          - container.image.tag
+          - container.id
+        labels:
+          - tag_name: kube_app_name
+            key: app.kubernetes.io/name
+            from: pod
+          - tag_name: kube_app_instance
+            key: app.kubernetes.io/instance
+            from: pod
+          - tag_name: kube_app_version
+            key: app.kubernetes.io/version
+            from: pod
+          - tag_name: kube_app_component
+            key: app.kubernetes.io/component
+            from: pod
+          - tag_name: kube_app_part_of
+            key: app.kubernetes.io/part-of
+            from: pod
+          - tag_name: kube_app_managed_by
+            key: app.kubernetes.io/managed-by
+            from: pod
+    resourcedetection:
+      detectors: [env, eks, ec2, system]
+      timeout: 2s
+      override: false
+    batch:
+      send_batch_max_size: 1000
+      send_batch_size: 100
+      timeout: 10s
```

Comment:
Whats the use of timeout in batch ? Would it result in dropping data?

Reply:
Its the _time duration after which a batch will be sent regardless of size._ See here: https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md. So it shouldn't drop any data.